types/HttpEnums.ts

```diff
@@ -0,0 +1,9 @@
+export enum HttpStatusCode {
```

Comment:
Is this used anywhere? I don't see any uses besides re-exporting in `types/Enums`.

Reply:
^